preprocessing/DataCleanser.py

```diff
@@ -3,14 +3,17 @@
 
 def dropNulls(df_covid):
     """
-    Removes any rows that contain a null value. We print out the abstract description here because we know there are
-    null value in there. We made the abstract a None type whenever no abstract was present in a JSON file.
+    Removes any rows that contain a null value. We currently create a string 'no abstract provided' if the abstract
+    is not present, so this method is to account for potential mishandled abstract entries.  For 'body_text'
+    entries that are empty strings, we convert them to NaN and then drop them.
 
     Should be completed before running other methods to reduce the number of rows examined.
 
     :param df_covid: Dataframe of covid data.
     :return: Data frame without any null values.
     """
+    nan_value = float("NaN")
+    df_covid['body_text'].replace("", nan_value, inplace=True)
```

Comment:
For this, can't you just do replace("", None, inplace=True) ? That would make it so don't need to worry about making the NaN value.

Reply:
Also, it may be good to use a regex to check for just whitespace too https://stackoverflow.com/a/21942746/7658632